### DISCUSSION TITLE
Is it possible retrieve from a JSON transcript?

### DISCUSSION THREAD

**Discussion by tonygenai** [FlowiseAI/Flowise#456] [Q&A]
Hello everyone,

I built a Conversational Retrieval QA Chain retrieving data from PDF files...super cool! Now I would like to do the same for a JSON transcript file, but I don't understand how to do it, I want also summarize the content of the JSON file.

Do you know how can do it please?
Thanks
Tony

**Comment by Kvkthecreator** [comment]
i haven't tried myself directly, but there is a document loader for json files. have you tried that one?

**Reply by tonygenai** [reply]
Thank you I will give a try
